src/KafkaW/ConsumerSettings.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+namespace KafkaW {
+
+class ConsumerSettings {
```

Comment:
Nothing against structs, but  I remember that some time ago was said to use classes over structures for uniformity. Don't know if the policy is still valid.

Reply:
LLVM standards seem to be fine with declaring classes with all public members as structs, I think it might cause compiler warnings if used with `-pedantic` as well